api/src/main/java/io/kafbat/ui/config/auth/LdapSecurityConfig.java

```diff
@@ -99,6 +107,10 @@ public DefaultLdapAuthoritiesPopulator ldapAuthoritiesExtractor(ApplicationConte
                                                                   AccessControlService acs) {
     var rbacEnabled = acs != null && acs.isRbacEnabled();
 
+    if (props.isActiveDirectory()) {
```

Comment:
Maybe it's worth doing this check at the very beginning of the method?

Reply:
Sure